### PR TITLE
test: Make mysql and kakfa integration tests locally runnable

### PR DESCRIPTION
**Pull request by hi-rustin**
### What problem does this PR solve? 

part of https://github.com/pingcap/ticdc/issues/2751

### What is changed and how it works?

add prepare integration test binaries script. We will use it in docker-compose.

usage: `docker-compose -f ./deployments/ticdc/docker-compose/docker-compose-kafka-integration.yml up -d --build`

### Check List 

Tests 

 - Manual test.
 
Code changes

None

Side effects

None

Related changes

None

### Release note 

```release-note
None
```

### CODE REVIEW

**Comment by hi-rustin** [comment]
@asddongmen @ben1009 

Could you please take a look? Thanks!

Ready for review!

**Review by overvenus — APPROVED** [review]
LGTM

**Review by asddongmen — APPROVED** [review]

**Comment by hi-rustin** [comment]
/assign @amyangfei 

Could you please help me trigger the merge process? Thanks!

File: deployments/ticdc/docker-compose/docker-compose-mysql-integration.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+      context: ../../../
+      dockerfile: ./deployments/ticdc/docker/integration-test.Dockerfile
+    depends_on:
+      - "zookeeper"
+      - "kafka"
+    command:
+      - bash
+      - -c
+      - make integration_test_mysql CASE=cdclog_s3
+    network_mode: "container:kafka"
```
Comment (hi-rustin) [inline deployments/ticdc/docker-compose/docker-compose-mysql-integration.yml:15]:
```suggestion
```

**Pull request merged**